packages/plugins/resize/src/index.ts

```diff
@@ -4,6 +4,14 @@
 import { addGondelPluginEventListener, getComponentByDomNode, GondelComponent } from "@gondel/core";
 import { INamespacedEventHandlerRegistry } from "@gondel/core/dist/GondelEventRegistry";
 
+export enum ResizeEvents {
```

Comment:
An enum produces unnecessary code which is not required in most cases.
In this example it will be more than 40% smaller for gondel-plugin and even more if used inside a project.

Maybe better:

```js
/**
 * The COMPONENT_RESIZED_EVENT event will be fired if a component size was changed because of a browser window resize
 */
export const COMPONENT_RESIZED_EVENT = "@gondel/plugin-resize--component-resized";
/**
 * The WINDOW_RESIZED_EVENT event will be fired if the browser window was resized
 */
export const WINDOW_RESIZED_EVENT = "@gondel/plugin-resize--window-resized";
```

Also using `/** ... */` instead of `//` will allow IDEs to provide a tooltip information if hovered over the variable.

Reply:
I had it first with two constants and switched to an enum in discussion with @janbiasi to be able to have a stronger type safety when using the event name as a method parameter..in that case should I switch back?

Reply:
Yes - don't see any benefits of a ENUM here - if we would need typing we could do sth like:

```js
export type RESIZE_EVENTS = typeof COMPONENT_RESIZED_EVENT | typeof WINDOW_RESIZED_EVENT;
```

That way it is also typed but still 40% smaller as type definitons are removed during compilations.

Reply:
ok, replacement done